parm/config/config.fv3

```diff
@@ -41,7 +41,9 @@ case $case_in in
         export layout_y=4
         export layout_x_gfs=2
         export layout_y_gfs=4
+        export npe_wav=70
         export nth_fv3=1
+        export nth_efcs=1
```

Comment:
why nth_efcs is needed here ?  config.fv3 is resolution dependent.   For a given resolution that the efts step is running with, the correct nth_fv3 is already set in config.fv3, which will be passed to nth_efcs in config.resources

Reply:
For running waves nth_fv3=4, which fails in efcs if nth_efcs is tied to nth_fv3. Decided to set nth_efcs separately. Am happy to discuss this change further since it's another important one.